### PR TITLE
Improvement/ebp 8 folder organisation

### PR DESCRIPTION
**Pull request by boosko80**

### CODE REVIEW

File: src/components/AlbumDeck/AlbumCard/Styled.tsx [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 const cssAlbumCardWithThumbnail = css<IStyledAlbumCardProps>`
   &:hover {
     background-image: url(${props => props.thumbnail});
     background-repeat: no-repeat;
   }
 
-  &:hover h3 {
+  &:hover ${StyledAlbumCardTitle} {
     color: white;
     background: rgba(0, 0, 0, 0.7);
```
Comment (Lroemon) [inline src/components/AlbumDeck/AlbumCard/Styled.tsx:20]: RGBA or Hex but need to choose :)

Comment (Lroemon) [inline src/components/AlbumDeck/AlbumCard/Styled.tsx:20]: rgba or hex :) ?

File: src/components/MainHero/Styled.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+const StyledHeroWrapper = styled.div`
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`
```
Comment (Lroemon) [inline src/components/MainHero/Styled.tsx:9]: Missing end ";" everywhere but you'll fix it with eslint

File: src/components/MainHero/index.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { StyledHeroWrapper, StyledHeroTitle } from "./Styled";
+import { NavigationBarWithConfig as NavigationBar } from "../NavigationBar";
+
+const MainHero = (): JSX.Element => {
```
Comment (Lroemon) [inline src/components/MainHero/index.tsx:4]: Why sometime the component is in the index.tsx and sometime inside ComponentName.tsx and then exported from index.tsx ?

File: src/components/NavigationBar/NavigationBar.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import { StyledNavigationBar, StyledNavigationLinks, StyledNavigationLink } from "./Styled";
+import { ILink } from "../../types/types";
+
+interface IProps {
+  navigationLinks: ILink[];
+}
+
+const NavigationBar = ({ navigationLinks }: IProps): JSX.Element => (
```
Comment (Lroemon) [inline src/components/NavigationBar/NavigationBar.tsx:8]: Here inside NavigationBar.tsx

File: src/components/NavigationBar/NavigationBarWithConfig.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { NavigationBar } from "./NavigationBar";
+import { linksConfig } from "../../config/navLinks";
+
+const NavigationBarWithConfig = (): JSX.Element => {
+  const { navigationLinks } = linksConfig;
+  return (
+    <NavigationBar navigationLinks={ navigationLinks } />
+  )
+  };
```
Comment (Lroemon) [inline src/components/NavigationBar/NavigationBarWithConfig.tsx:9]: Looking forward for eslint !

File: src/pages/index.tsx [hunk 6]
```diff
@@ -0,0 +1,6 @@
+export { AboutPage } from "./AboutPage";
```
Comment (Lroemon) [inline src/pages/index.tsx:1]: Is this file usefull ?

**Review by Lroemon — CHANGES_REQUESTED** [review]

File: src/pages/index.tsx (hunk 6 above)
Comment (Lroemon) [reply src/pages/index.tsx:1]: "Is the index here"*

File: src/components/NavigationBar/Styled.tsx [hunk 7]
```diff
@@ -1,5 +1,17 @@
-import { NavLink } from "react-router-dom";
 import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+
+interface IStyledLinks {
+  length: number;
+}
+
+const StyledNavigationBar = styled.nav``;
```
[the remark this answers is not in the document]
Comment (boosko80) [reply src/components/NavigationBar/Styled.tsx:8]: <code><nav></code> ?

**Review by Lroemon — APPROVED** [review]

**Pull request merged**